test/e2e/updaterun_test.go

```diff
@@ -24,10 +24,8 @@ import (
 )
 
 const (
-	// The current stage wait duration is 1 minute.
-	// It might not be enough to wait for the updateRun to reach the wanted state with existing eventuallyDurtation, 2 minutes.
-	// Here we extend it to 3 minutes.
-	updateRunEventuallyDuration = time.Minute * 3
+	// The current stage wait between clusters are 15 seconds
```

Comment:
stage wait time is 1 minute. While cluster wait time is 15 seconds: https://github.com/Azure/fleet/blob/main/pkg/controllers/updaterun/execution.go#L34.

Looks like the e2e still fails because the duration is not long enough.

Reply:
We are not really waiting 1 min between stages. Plase see the wait time fix, that was the main reason there were so much delays needed in the e2e between stages.